examples/interfaces/IAccessControl/IAccessControl.rs

```diff
@@ -0,0 +1,56 @@
+// Generated with Sol2Ink v0.2.4
+// https://github.com/Supercolony-net/sol2ink
+
+use brush::traits::AccountId;
+
+#[ink(event)]
+pub struct RoleAdminChanged {
+	#[ink(topic)]
+	role: [u8; 32],
+	#[ink(topic)]
+	previous_admin_role: [u8; 32],
+	#[ink(topic)]
+	new_admin_role: [u8; 32],
+}
+
+#[ink(event)]
+pub struct RoleGranted {
+	#[ink(topic)]
+	role: [u8; 32],
+	#[ink(topic)]
+	account: AccountId,
+	#[ink(topic)]
+	sender: AccountId,
+}
+
+#[ink(event)]
+pub struct RoleRevoked {
+	#[ink(topic)]
+	role: [u8; 32],
+	#[ink(topic)]
+	account: AccountId,
+	#[ink(topic)]
+	sender: AccountId,
+}
+
+#[brush::wrapper]
+pub type IAccessControlRef = dyn IAccessControl;
+
+#[brush::trait_definition]
+pub trait IAccessControl {
+	#[ink(message)]
```

Comment:
Is that possible to parse comments and put them here?

Reply:
I will do that later